src/languages/en.ts

```diff
@@ -562,6 +562,7 @@ export default {
         threadRequestReportName: ({formattedAmount, comment}: ThreadRequestReportNameParams) => `${formattedAmount} request${comment ? ` for ${comment}` : ''}`,
         threadSentMoneyReportName: ({formattedAmount, comment}: ThreadSentMoneyReportNameParams) => `${formattedAmount} sent${comment ? ` for ${comment}` : ''}`,
         tagSelection: ({tagName}: TagSelectionParams) => `Select a ${tagName} to add additional organization to your money`,
+        categorySelection: 'Select a category to add additional organization to your money',
```

Comment:
I don't think this copy was checked? It does not really read like a proper English sentence. We're going to create an issue to modify it.

Reply:
Fixed here: https://github.com/Expensify/App/pull/38242